Add copy/paste commands and menu items to chat view

planger opened the pull request:
Fixes https://github.com/eclipsesource/osweek-2024/issues/119

sdirix reviewed (changes requested):
- When I click `Copy` and nothing is selected, then nothing is copied. We should either copy the current line or message or not enable/show the copy action then. For reference:
  - Within the about dialog of Theia, copy is not shown if nothing is selected (normal browser context window).
  - Within Monaco `Copy` will copy the line where the mouse was when nothing was selected
- `Copy All` only copies the message in which `Copy All` was executed. I would have expected the whole history to be copied. I would advocate to rename to `Copy message` if this behavior is intended
- `Copy All` always appends a comma `,` at the end of the message.

planger commented:
@sdirix Thanks for the thorough review. I've addressed the comments.

I think just removing `Copy` is a bit lazy, so I left it in, but test whether there is a selection, if yes just the selection is copied, if no, the entire message is copied. I also kept `Copy Message` even though it technically is the same as `Copy` without selection, but it is a bit clearer. Also I've added `Copy All` which copies the entire thread.

I decided to go the extra mile here as copying from the chat is imho a very frequent use case, having all options at your finger tips will hopefully improve the experience.

@JonasHelming Yes, but this mostly worked already before.

JonasHelming commented:
@planger This does not fix https://github.com/eclipsesource/osweek-2024/issues/116 correct?
(It is a separate issue, just wanted to check)
